repo/packages/O/openldap/0/marathon.json.mustache

```diff
@@ -0,0 +1,81 @@
+{
+  "id": "{{service.name}}",
+  "cpus": {{openldap.cpus}},
+  "mem": {{openldap.mem}},
+  "instances": 1,
+  "env": {
+    "SLAPD_PASSWORD": "{{openldap.admin_password}}",
+    "SLAPD_DOMAIN": "{{openldap.domain}}",
+    "SLAPD_ORGANIZATION": "{{openldap.organization}}",
+    "SLAPD_CONFIG_PASSWORD": "{{openldap.config_password}}",
+    "SLAPD_ADDITIONAL_SCHEMAS": "{{openldap.additional_schemas}}",
+    "SLAPD_ADDITIONAL_MODULES": "{{openldap.additional_modules}}",
+    "SLAPD_FORCE_RECONFIGURE": "{{openldap.force_reconfigure}}"
+  },
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "{{resource.assets.container.docker.openldap-docker}}",
+      "forcePullImage": false,
+      "network": "BRIDGE",
+      "portMappings": [
+        {
+          "containerPort": 389,
+          "hostPort": 0,
+          "servicePort": {{openldap.port}},
+          "protocol": "tcp",
+          "name": "{{service.name}}",
+          "labels": {
+            "VIP_0": "/{{service.name}}:389"
+          }
+        },
+        {
+          "containerPort": 636,
+          "hostPort": {{openldap.port_ssl}},
+          "servicePort": {{openldap.port_ssl}},
+          "protocol": "tcp",
+          "name": "{{service.name}}-ssl",
+          "labels": {
+            "VIP_1": "/{{service.name}}-ssl:636"
+          }
+        }
+      ]
+    }
+  },
+  "healthChecks": [
+    {
+      "protocol": "COMMAND",
+      "command": {
+        "value": "ps -Aef | grep slapd"
+      },
+      "gracePeriodSeconds": 300,
+      "intervalSeconds": 60,
+      "timeoutSeconds": 20,
+      "maxConsecutiveFailures": 3
+    }
+  ],
+  "constraints": [
+    [
+      "hostname",
+      "UNIQUE"
+    ]
+  ],
+  "volumes": [
+    {
+      "containerPath": "/var/lib/ldap",
+      "hostPath": "{{openldap.host_volume}}/{{openldap.name}}/ldap",
+      "mode": "RW"
+    },
+    {
+      "containerPath": "/etc/ldap/slapd.d",
+      "hostPath": "{{openldap.host_volume}}/{{openldap.name}}/slapd.d",
+      "mode": "RW"
+    }
+  ],
+  "labels": {
+    "DCOS_SERVICE_NAME": "{{service.name}}",
+    "HAPROXY_0_VHOST": "{{openldap.virtual_host}}",
```

Comment:
Why would I want my ldap available without auth externally? Shouldn't this be going through adminrouter?